etc/kayobe/pulp-repo-versions.yml

```diff
@@ -34,3 +34,17 @@ stackhpc_pulp_repo_rocky_8_7_extras_version: 20221201T192704
 stackhpc_pulp_repo_rocky_8_7_nfv_version: 20221202T032715
 stackhpc_pulp_repo_rocky_8_7_powertools_version: 20221202T032715
 stackhpc_pulp_repo_mlnx_ofed_5_7_1_0_2_0_rhel8_6_version: 20220920T151419
+stackhpc_pulp_repo_rocky_9_appstream_version: 20230228T044432
```

Comment:
I ran the CI job to generate this file and it came up with this ordering (only want to update the new repos):

```
---
# Do not edit! This file is autogenerated by Ansible.
stackhpc_pulp_repo_centos_stream_8_advanced_virtualization_version: 20211122T102435
stackhpc_pulp_repo_centos_stream_8_appstream_version: 20230211T050436
stackhpc_pulp_repo_centos_stream_8_baseos_version: 20230211T050436
stackhpc_pulp_repo_centos_stream_8_extras_version: 20220401T032901
stackhpc_pulp_repo_centos_stream_8_nfv_extras_version: 20220609T110556
stackhpc_pulp_repo_centos_stream_8_nfv_openvswitch_version: 20230302T031902
stackhpc_pulp_repo_centos_stream_8_openstack_yoga_version: 20230228T044432
stackhpc_pulp_repo_centos_stream_8_opstools_version: 20230301T034123
stackhpc_pulp_repo_centos_stream_8_powertools_version: 20230211T050436
stackhpc_pulp_repo_centos_stream_8_storage_ceph_pacific_version: 20230201T025809
stackhpc_pulp_repo_centos_stream_9_docker_version: 20230228T044432
stackhpc_pulp_repo_centos_stream_9_nfv_openvswitch_version: 20230302T031902
stackhpc_pulp_repo_centos_stream_9_opstools_version: 20230301T034123
stackhpc_pulp_repo_centos_stream_9_storage_ceph_quincy_version: 20230228T044432
stackhpc_pulp_repo_docker_ce_ubuntu_version: 20220708T132615
stackhpc_pulp_repo_docker_version: 20230217T091757
stackhpc_pulp_repo_elasticsearch_logstash_kibana_7_x_version: 20230203T025251
stackhpc_pulp_repo_epel_9_version: 20230302T031902
stackhpc_pulp_repo_epel_modular_version: 20220913T043117
stackhpc_pulp_repo_epel_version: 20230301T034123
stackhpc_pulp_repo_grafana_version: 20230302T031902
stackhpc_pulp_repo_mariadb_10_6_centos8_version: 20230206T150339
stackhpc_pulp_repo_mlnx_ofed_5_7_1_0_2_0_rhel8_6_version: 20220920T151419
stackhpc_pulp_repo_rabbitmq_erlang_version: 20230210T031947
stackhpc_pulp_repo_rabbitmq_server_version: 20230213T030451
stackhpc_pulp_repo_rhel_9_influxdb_version: 20230228T044432
stackhpc_pulp_repo_rhel_9_mariadb_10_6_version: 20230228T044432
stackhpc_pulp_repo_rhel_9_mlnx_ofed_5_9_0_5_6_0_version: 20230228T044432
stackhpc_pulp_repo_rhel_9_treasuredata_4_version: 20230228T044432
stackhpc_pulp_repo_rocky_8_6_appstream_version: 20221105T035018
stackhpc_pulp_repo_rocky_8_6_baseos_version: 20221105T035018
stackhpc_pulp_repo_rocky_8_6_extras_version: 20221105T035018
stackhpc_pulp_repo_rocky_8_6_nfv_version: 20221103T030410
stackhpc_pulp_repo_rocky_8_6_powertools_version: 20221109T032314
stackhpc_pulp_repo_rocky_8_7_appstream_version: 20230224T030859
stackhpc_pulp_repo_rocky_8_7_baseos_version: 20230221T033857
stackhpc_pulp_repo_rocky_8_7_extras_version: 20230217T091757
stackhpc_pulp_repo_rocky_8_7_nfv_version: 20230223T031000
stackhpc_pulp_repo_rocky_8_7_powertools_version: 20230301T034123
stackhpc_pulp_repo_rocky_9_appstream_version: 20230228T044432
stackhpc_pulp_repo_rocky_9_baseos_version: 20230228T044432
stackhpc_pulp_repo_rocky_9_crb_version: 20230228T044432
stackhpc_pulp_repo_rocky_9_extras_version: 20230228T044432
stackhpc_pulp_repo_rocky_9_highavailability_version: 20230228T044432
stackhpc_pulp_repo_treasuredata_4_version: 20221105T035018
stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20220804T040153
stackhpc_pulp_repo_ubuntu_focal_security_version: 20220804T040153
stackhpc_pulp_repo_ubuntu_focal_version: 20220804T040153
```
https://github.com/stackhpc/stackhpc-release-train/actions/runs/4314647141

Reply:
I've matched this order now